include/lsst/afw/table/Schema.h

```diff
@@ -46,7 +46,7 @@ class Schema {
 public:
 
     // This variable is defined in SchemaImpl, but is replicated here as
-    // a static so that it is available to SWIG.
+    // a static so that it is available to Python.
     static int const VERSION = detail::SchemaImpl::VERSION;
```

Comment:
Also, it could be constexpr, but I don't think there's much value for it to be, and I'd rather not change the pure C++ more than I have to.